build.sbt

```diff
@@ -231,19 +231,32 @@ lazy val nir =
     .settings(mavenPublishSettings)
     .dependsOn(util)
 
+lazy val nirparser =
+  project
+    .in(file("nirparser"))
+    .settings(toolSettings)
+    .settings(noPublishSettings)
+    .settings(
+      libraryDependencies ++= Seq(
+        "com.lihaoyi" %% "fastparse"  % "1.0.0",
+        "com.lihaoyi" %% "scalaparse" % "1.0.0",
+        compilerPlugin(
+          "org.scalamacros" % "paradise" % "2.1.0" cross CrossVersion.full),
+        "org.scalacheck" %% "scalacheck" % "1.13.4" % "test",
+        "org.scalatest"  %% "scalatest"  % "3.0.0"  % "test"
+      )
+    )
+    .dependsOn(nir)
+
 lazy val tools =
   project
     .in(file("tools"))
     .settings(toolSettings)
     .settings(mavenPublishSettings)
     .settings(
       libraryDependencies ++= Seq(
-        "com.lihaoyi"    %% "fastparse"  % "0.4.2",
-        "com.lihaoyi"    %% "scalaparse" % "0.4.2",
         "org.scalacheck" %% "scalacheck" % "1.13.4" % "test",
-        compilerPlugin(
-          "org.scalamacros" % "paradise" % "2.1.0" cross CrossVersion.full),
-        "org.scalatest" %% "scalatest" % "3.0.0" % "test"
```

Comment:
I think that you can also remove this from `project/build.sbt`:

https://github.com/scala-native/scala-native/blob/682dab82c7c689d00ebe6d263685694994287ad4/project/build.sbt#L17-L20

Reply:
Indeed. 👍

Reply:
a6e6257